serialization/physics.proto

```diff
@@ -116,13 +116,29 @@ message HierarchicalSystem {
 
 message KeplerianElements {
   required Frame frame = 1;
-  required double eccentricity = 2;
+  optional double eccentricity = 2;
```

Comment:
As mentioned elsewhere I think we should have a bit more structure here instead of 25 flat optional fields.  At a minimum, comments explaining the different sections/groups.